tools/gen_usb_descriptor.py

```diff
@@ -76,7 +80,7 @@ def strings_in_order(cls):
     description="CDC comm",
     bInterfaceClass=cdc.CDC_CLASS_COMM,  # Communications Device Class
     bInterfaceSubClass=cdc.CDC_SUBCLASS_ACM,  # Abstract control model
-    bInterfaceProtocol=cdc.CDC_PROTOCOL_NONE,
+    bInterfaceProtocol=cdc.CDC_PROTOCOL_V25TER,
```

Comment:
`CDC_PROTOCOL_V25TER` means it implements AT commands, so I changed it to `CDC_PROTOCOL_NONE` when I was working on this. But did you find a problem with `CDC_PROTOCOL_NONE`?

Reply:
TinyUSB didn't match the interface to its driver but I'll update it to be less strict.